data/resources/resources.gresource.xml

```diff
@@ -3,6 +3,10 @@
   <gresource prefix="/com/github/melix99/telegrand/">
     <file preprocess="xml-stripblanks">icons/scalable/actions/phone-oldschool-symbolic.svg</file>
     <file preprocess="xml-stripblanks">icons/scalable/actions/send-symbolic.svg</file>
+    <file preprocess="xml-stripblanks">icons/scalable/status/message-failed-symbolic.svg</file>
+    <file preprocess="xml-stripblanks">icons/scalable/status/message-pending-symbolic.svg</file>
+    <file preprocess="xml-stripblanks">icons/scalable/status/message-read-symbolic.svg</file>
+    <file preprocess="xml-stripblanks">icons/scalable/status/message-unread-right-symbolic.svg</file>
```

Comment:
```suggestion
    <file preprocess="xml-stripblanks">icons/scalable/status/message-unread-symbolic.svg</file>
```

I think I understand why you called it that way, but I think it's a confusing name, so it's fine just calling it without the "right" suffix.